src/views/pages/HomeView/Hero.js

```diff
@@ -53,30 +51,50 @@ const useStyles = makeStyles(theme => ({
 
 function Hero({ className, ...rest }) {
   const classes = useStyles();
-
   return (
-    <div className={clsx(classes.root, className)} {...rest}>
-      <Container maxWidth="lg">
-        <Grid container spacing={3}>
-          <Grid item xs={12} md={6}>
+    <div
```

Comment:
@AdityaTeltia See the website with around 70% zoom, the blob and text is nowhere there. This is due to the container you have removed. If you put your code inside containers than those things are automatically managed. 

Since most of the monitors now a days follows this 67%, 80% default zoom ratio, it is important to not break the UI there.
![image](https://user-images.githubusercontent.com/48255244/108587077-98652500-7377-11eb-94b3-e15a16d9b6f3.png)

Reply:
Also , I have not removed ```Container``` . Every component is the same it just may be the line number changed so it is showing in red.

Reply:
![causefolio](https://user-images.githubusercontent.com/67232537/108682344-2f67e380-7516-11eb-888c-a150cae74715.JPG)
What about this at 70% zoom ?

Reply:
@AdityaTeltia, don't we have a full blob? I mean the whole rounded one.
This is looking much better now, btw.

Reply:
No its not a complete blob , 
![blob](https://user-images.githubusercontent.com/67232537/108684633-24628280-7519-11eb-93d3-903f98b81d38.png)